tests/data/oz_proxy_compiled_contract.json

```diff
@@ -0,0 +1 @@
lls.SendMessageToL1SysCall","members":{"payload_ptr":{"cairo_type":"felt*","offset":3},"payload_size":{"cairo_type":"felt","offset":2},"selector":{"cairo_type":"felt","offset":0},"to_address":{"cairo_type":"felt","offset":1}},"size":4,"type":"struct"},"starkware.starknet.common.syscalls.StorageRead":{"full_name":"starkware.starknet.common.syscalls.StorageRead","members":{"request":{"cairo_type":"starkware.starknet.common.syscalls.StorageReadRequest","offset":0},"response":{"cairo_type":"starkware.starknet.common.syscalls.StorageReadResponse","offset":2}},"size":3,"type":"struct"},"starkware.starknet.common.syscalls.StorageReadRequest":{"full_name":"starkware.starknet.common.syscalls.StorageReadRequest","members":{"address":{"cairo_type":"felt","offset":1},"selector":{"cairo_type":"felt","offset":0}},"size":2,"type":"struct"},"starkware.starknet.common.syscalls.StorageReadResponse":{"full_name":"starkware.starknet.common.syscalls.StorageReadResponse","members":{"value":{"cairo_type":"felt","offset":0}},"size":1,"type":"struct"},"starkware.starknet.common.syscalls.StorageWrite":{"full_name":"starkware.starknet.common.syscalls.StorageWrite","members":{"address":{"cairo_type":"felt","offset":1},"selector":{"cairo_type":"felt","offset":0},"value":{"cairo_type":"felt","offset":2}},"size":3,"type":"struct"},"starkware.starknet.common.syscalls.TxInfo":{"full_name":"starkware.starknet.common.syscalls.TxInfo","members":{"account_contract_address":{"cairo_type":"felt","offset":1},"chain_id":{"cairo_type":"felt","offset":6},"max_fee":{"cairo_type":"felt","offset":2},"signature":{"cairo_type":"felt*","offset":4},"signature_len":{"cairo_type":"felt","offset":3},"transaction_hash":{"cairo_type":"felt","offset":5},"version":{"cairo_type":"felt","offset":0}},"size":7,"type":"struct"},"starkware.starknet.common.syscalls.delegate_call":{"decorators":[],"pc":3,"type":"function"},"starkware.starknet.common.syscalls.delegate_call.Args":{"full_name":"starkware.starknet.common.syscalls.delegate_call.Args","members":{"calldata":{"cairo_type":"felt*","offset":3},"calldata_size":{"cairo_type":"felt","offset":2},"contract_address":{"cairo_type":"felt","offset":0},"function_selector":{"cairo_type":"felt","offset":1}},"size":4,"type":"struct"},"starkware.starknet.common.syscalls.delegate_call.ImplicitArgs":{"full_name":"starkware.starknet.common.syscalls.delegate_call.ImplicitArgs","members":{"syscall_ptr":{"cairo_type":"felt*","offset":0}},"size":1,"type":"struct"},"starkware.starknet.common.syscalls.delegate_call.Return":{"full_name":"starkware.starknet.common.syscalls.delegate_call.Return","members":{"retdata":{"cairo_type":"felt*","offset":1},"retdata_size":{"cairo_type":"felt","offset":0}},"size":2,"type":"struct"},"starkware.starknet.common.syscalls.delegate_call.SIZEOF_LOCALS":{"type":"const","value":0},"starkware.starknet.common.syscalls.delegate_call.__temp0":{"cairo_type":"felt","full_name":"starkware.starknet.common.syscalls.delegate_call.__temp0","references":[{"a
```

Comment:
can't we compile it from source?

Reply:
OZ Proxy preset imports various utilities from cairo-contracts. I copied this file from Starknet.py and minimized it.

Reply:
sounds like a good reason to keep built file.

also this makes tests faster.